### DISCUSSION TITLE
Video: Coding on the iPad with code-server Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by bpmct** [coder/code-server#3156] [Show and tell]
For anyone looking to set up a Raspberry Pi 4 with code-server and connect locally via your iPad, here's an awesome tutorial by and guide I found!

YouTube: https://www.youtube.com/watch?v=PoxumlaKtyI
GitHub: https://github.com/andreivisan/Raspberry-Pi-4-Code-Server

**Comment by andreivisan** [comment]
I also created a video and docs on how to upgrade code-server on Raspberry Pi 4 

YouTube: https://youtu.be/75Sn-fGt3UA
GitHub: https://github.com/andreivisan/Raspberry-Pi-4-Code-Server
